packages/victory-core/package.json

```diff
@@ -122,6 +130,40 @@
         "../victory-vendor:build:lib:esm"
       ]
     },
+    "types:check": {
+      "command": "nps types:pkg:check",
+      "files": [
+        "src/**/*.{ts,tsx}",
+        "../../tsconfig.base.json",
+        "tsconfig.json"
+      ],
+      "dependencies": [
+        "../victory-vendor:types:create",
+        "../victory-axis:types:create",
+        "../victory-bar:types:create",
+        "types:create",
+        "../victory-line:types:create"
+      ],
+      "output": []
+    },
+    "types:create": {
+      "command": "nps types:pkg:create",
```

Comment:
Does tsc depend on lib/es ts files being present and built first?

Reply:
Tsc does not care about the built es/cjs `*.js` files. It only cares about `.d.ts` files.
So the dependency chain only lists `types:create` tasks